tests/Command/ExportListCommandsCommandTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Tests\Command;
+
+use App\Command\ExportListCommandsCommand;
+use App\Tests\TestCase;
+use Draw\Component\Tester\Application\CommandDataTester;
+use Draw\Component\Tester\Application\CommandTestTrait;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * @covers \App\Command\ExportListCommandsCommand
+ */
+class ExportListCommandsCommandTest extends TestCase
+{
+    use CommandTestTrait;
+
+    public function createCommand(): Command
+    {
+        return static::getService(ExportListCommandsCommand::class);
```

Comment:
getService should be deprecated. Use static::getContainer()->get(ExportListCommandsCommand::class);